crates/bevy_app/src/app.rs

```diff
@@ -86,7 +86,7 @@ impl App {
         }
     }
 
-    /// Advances the execution of the [`Schedule`] by one cycle.
+    /// Advances the execution of the [`Schedule`] by one cycle and updates sub apps.
```

Comment:
IMO this added line should go in the body, rather than the tooltip.

It's not essential information, but is a useful detail.